crates/pathfinder/src/rpc.rs

```diff
@@ -3047,13 +3048,13 @@ mod tests {
                     DeclareTransactionResult {
                         transaction_hash: StarknetTransactionHash(
                             StarkHash::from_hex_str(
-                                "0x05aa2a3468bfe8942e321dde6fbebc2c0359da311340c7e35b0bb9b089d4d469"
+                                "0x077ccba4df42cf0f74a8eb59a96d7880fae371edca5d000ca5f9985652c8a8ed"
```

Comment:
I assume all of these are reply values which have now changed with the feeder_gateway. Please correct if I misunderstand.

Reply:
Exactly. These hashes are calculated using the ABI, so now that we're actually passing an ABI to the sequencer they have changed.